index.js

```diff
@@ -330,7 +340,32 @@ function redisStore(args) {
       if (err) {
         return cb && cb(err);
       }
-      conn.keys(pattern, handleResponse(conn, cb));
+
+      // Use an object to dedupe as scan can return duplicates
+      var keysObj = {};
+      var count = 100;
+
+      (function nextBatch(cursorId) {
```

Comment:
I guess it's to start the execution of the scan, then it's called recursively until there is no next, isn't it @jeff-kilbride ?